src/app/user-page/user-page.component.html

```diff
@@ -43,6 +43,16 @@ <h4 matLine class="accent-1-dark mt-2 mb-0">{{ user?.username }}</h4>
                   <span>{{ gitHubProfile.username }}</span>
                 </a>
               </span>
+              <span *ngIf="loggedInUserIsAdminOrCurator">
+                <button
```

Comment:
I know this is only used by our own admins/curators, but it might be helpful to add a tooltip explaining what this button does like, `View the user's GitHub App Logs`, since `Apps Logs` is pretty non-descriptive when it's on the users page